SeaSharpe_CVGS/SeaSharpe_CVGS/Migrations/201512121823090_Schema_Additions.cs

```diff
@@ -8,7 +8,7 @@ public partial class Schema_Additions : DbMigration
         public override void Up()
         {
             AddColumn("dbo.Members", "Preferences", c => c.String(maxLength: 500));
-            AddColumn("dbo.Games", "IsActive", c => c.Boolean(nullable: false));
+            AddColumn("dbo.Games", "IsActive", c => c.Boolean(nullable: false, defaultValue: true));
```

Comment:
If this isn't working we might want to take it out. Just in case it breaks something else somehow.